pkg/cmd/kafka/update/update.go

```diff
@@ -0,0 +1,224 @@
+				var missingFlags []string
+				if opts.owner == "" {
+					missingFlags = append(missingFlags, "owner")
+				}
+				if !opts.skipConfirm {
+					missingFlags = append(missingFlags, "yes")
+				}
+				if len(missingFlags) > 0 {
+					return flag.RequiredWhenNonInteractiveError(missingFlags...)
+				}
+			}
+
+			validOutputFormats := flagutil.ValidOutputFormats
+			if opts.outputFormat != "" && !flagutil.IsValidInput(opts.outputFormat, validOutputFormats...) {
+				return flag.InvalidValueError("output", opts.outputFormat, validOutputFormats...)
+			}
+
+			if opts.name != "" && opts.id != "" {
+				return errors.New(opts.localizer.MustLocalize("service.error.idAndNameCannotBeUsed"))
+			}
+
+			if opts.id != "" || opts.name != "" {
+				return run(&opts)
+			}
+
+			cfg, err := opts.Config.Load()
+			if err != nil {
+				return err
+			}
+
+			var kafkaConfig *config.KafkaConfig
+			if cfg.Services.Kafka == kafkaConfig || cfg.Services.Kafka.ClusterID == "" {
+				return errors.New(opts.localizer.MustLocalize("kafka.common.error.noKafkaSelected"))
+			}
+
+			opts.id = cfg.Services.Kafka.ClusterID
+
+			return run(&opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.outputFormat, "output", "o", dump.JSONFormat, opts.localizer.MustLocalize("kafka.common.flag.output.description"))
+	cmd.Flags().StringVar(&opts.id, "id", "", opts.localizer.MustLocalize("kafka.update.flag.id"))
+	cmd.Flags().StringVar(&opts.owner, "owner", "", opts.localizer.MustLocalize("kafka.update.flag.owner"))
+	cmd.Flags().BoolVarP(&opts.skipConfirm, "yes", "y", false, opts.localizer.MustLocalize("kafka.update.flag.yes"))
+	cmd.Flags().StringVar(&opts.name, "name", "", opts.localizer.MustLocalize("kafka.update.flag.name"))
+
+	_ = cmd.MarkFlagRequired("owner")
+
+	_ = kafkacmdutil.RegisterNameFlagCompletionFunc(cmd, f)
+
+	return cmd
+}
+
+func run(opts *Options) error {
+	connection, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	api := connection.API()
+
+	var kafkaInstance *kafkamgmtclient.KafkaRequest
+	ctx := context.Background()
+	if opts.name != "" {
+		kafkaInstance, _, err = kafka.GetKafkaByName(ctx, api.Kafka(), opts.name)
+		if err != nil {
+			return err
+		}
+	} else {
+		kafkaInstance, _, err = kafka.GetKafkaByID(ctx, api.Kafka(), opts.id)
+		if err != nil {
+			return err
+		}
+	}
+
+	if opts.owner == kafkaInstance.GetOwner() {
+		opts.logger.Info(opts.localizer.MustLocalize("kafka.update.log.info.sameOwnerNoChanges", localize.NewEntry("Owner", opts.owner), localize.NewEntry("InstanceName", kafkaInstance.GetName())))
+		return nil
+	}
+
+	updateObj := kafkamgmtclient.NewKafkaUpdateRequest(opts.owner)
+
+	// create a text block with a summary of what is being updated
+	updateSummary := generateUpdateSummary(reflect.ValueOf(*updateObj), reflect.ValueOf(*kafkaInstance))
+
+	opts.logger.Infof(`
+ %v 🗒️
+
+ %v`, color.Underline(color.Bold(opts.localizer.MustLocalize("kafka.update.summaryTitle"))), updateSummary)
```

Comment:
```suggestion
	opts.logger.Infof("\n%v 🗒️ \n \n %v", color.Underline(color.Bold(opts.localizer.MustLocalize("kafka.update.summaryTitle"))), updateSummary)
```

Reply:
Formatting is slightly different to your suggestion. I like the multiline string as it is easier to visualize how the formatting will be. Are there any reasons I should not?